src/test/java/bio/terra/service/snapshot/SnapshotServiceTest.java

```diff
@@ -118,7 +122,18 @@ public void testRetrieveSnapshot() {
                                                 .cloudResource(
                                                     GoogleCloudResource.BUCKET.toString())
                                                 .cloudPlatform(CloudPlatform.GCP))))))
-                .tables(List.of(new TableModel().name(SNAPSHOT_TABLE_NAME).primaryKey(List.of())))
+                .tables(
+                    List.of(
+                        new TableModel()
+                            .name(SNAPSHOT_TABLE_NAME)
+                            .primaryKey(List.of())
+                            .columns(
+                                List.of(
+                                    new ColumnModel()
```

Comment:
If there was a unit test for `makeColumnModelFromColumn()` (and the other similar methods in `SnapshotService`), I think it would make it easier to read and update the test code, rather than adding more data to this test case.